.github/workflows/CreateWidgetRelease.yml

```diff
@@ -0,0 +1,38 @@
+name: Create widget release
+
+on:
+    workflow_dispatch:
+        inputs:
+            widget-name:
+                description: "Widget to release"
+                required: true
+
+jobs:
+    release:
+        name: "Create widget release"
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checking-out code"
+              uses: actions/checkout@v2
+              with:
+                  submodules: false
+            - name: "Installing dependencies"
+              run: npm install
+            - name: "Building ${{ github.event.inputs.widget-name }} widget"
+              run: "npm run release -- --scope '${{ steps.scope.outputs._0 }}'"
```

Comment:
We dont have a scope step here to use this variable.

Reply:
Oops, copied that from the marketplace one. Fixed it to use the github workflow input